build/yamls/base/agent-rbac.yml

```diff
@@ -15,6 +15,7 @@ rules:
     resources:
       - nodes
       - pods
```

Comment:
@tnqn @antoninbas : do you know if Agent needs to watch and list Pods? I did not find the code.

Reply:
https://github.com/vmware-tanzu/antrea/blob/master/pkg/agent/cniserver/server.go#L602

we could consider changing this code though if there is a good reason to. IIRC, this is just used to sanity check the interface store we retrieve from OVSDB.

Reply:
Ok. At least it does not watch right. Do you think worthwhile to remove watch from Pods and Services?

Reply:
Yes, I think you can split the resources and only provide the "list" verb for Pods

I guess they were grouped for convenience?